packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -386,7 +388,9 @@ async function loadContracts(type?: ContractType) {
     contracts.value = [...nodeContracts.value, ...nameContracts.value, ...rentContracts.value];
 
     // Update the total cost of the contracts.
-    await getTotalCost();
+    if (!type) {
+      await getTotalCost();
+    }
```

Comment:
can you explain this change, please? I think the get total cost should be triggered on mount, and on refresh after delete contract, and we should ignore all other actions

Reply:
To avoid calling the `getTotalCost()` on unwanted cases like using pagination or sorting, I see the type isn't passed in several cases like the `refresh` button and `onDeletedContracts()` so I used this condition